setup_cfg_fmt.py

```diff
@@ -316,6 +316,44 @@ def _is_ok_classifier(s: str) -> bool:
     return [s for s in classifiers if _is_ok_classifier(s)]
 
 
+def _imp_classifiers(setup_cfg: str) -> Optional[str]:
+    tox_config_to_names = {
+        'py': 'CPython',
+        'pypy': 'PyPy',
+        'jython': 'Jython',
+    }
+
+    cfg = configparser.ConfigParser()
+    cfg.read(setup_cfg)
+    classifiers = cfg.get('metadata', 'classifiers', fallback='')
+    tags = []
```

Comment:
instead of re-reading here, I'd pass in `cfg` from the calling function (the others in the file do similar)